sdk/storage/tests_invoke.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  EnvVars: {}
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      BuildTargetingString: azure-storage-blob
+      JobName: Test_Blob
+      ServiceDirectory: storage
+      EnvVars: ${{ parameters.EnvVars }}
+
+
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      BuildTargetingString: azure-storage-queue
+      JobName: Test_Queue
+      ServiceDirectory: storage
+      EnvVars: ${{ parameters.EnvVars }}
+
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      MaxParallel: 1
```

Comment:
thanks for making datalake test sequential

Reply:
Why can the datalake tests not be run in parallel?  Is there a plan to fix this in the future?

Reply:
datalake is still using the old framework, because the new one is not very friendly to developers when new features is added 😀 So datalake will be migrated to the new test framework when all test problems are addressed for example https://github.com/Azure/azure-sdk-for-python/pull/8930